Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -381,17 +377,10 @@ static inline FilterOperations blendFunc(const FilterOperations& from, const Fil
     return result;
 }
 
-static inline RefPtr<StyleImage> blendFilter(CachedImage* image, const FilterOperations& from, const FilterOperations& to, const CSSPropertyBlendingContext& context)
+static inline RefPtr<StyleImage> blendFilter(RefPtr<StyleImage> inputImage, const FilterOperations& from, const FilterOperations& to, const CSSPropertyBlendingContext& context)
```

Comment:
At the moment, I can't prove that inputImage is non-null due to the parser currently allowing using `none` for images in some cases. This is something I want to improve if possible, since the specs don't actually allow this and it doesn't generally seem useful to be able to do.